### ISSUE TITLE
[Bug]: ftbquests crash

### ISSUE THREAD

**Issue by BlackAures1** [FTBTeam/FTB-Mods-Issues#508]
### Mod

 ftb quests

### Mod version

ftb-quests-forge-1802.3.8-build.101

### Forge / Fabric version

1.18.2 40.1.80

### Modpack & version

https://www.curseforge.com/minecraft/modpacks/play-as-dragon

### What issue are you having?

Hello. When opening quests my players began to have game crashes. The crash report mentions your mod. Can you help with this?

### Crashlogs

[crash-2022-10-20_14.22.36-client.txt](https://github.com/FTBTeam/FTB-Mods-Issues/files/9833408/crash-2022-10-20_14.22.36-client.txt)

### Steps to reproduce

Happens simply when you open the quest panel (J). Personally, I can not repeat this bug, but the person who sent the crashreport got it two times in a row.

![image](https://user-images.githubusercontent.com/43911230/197042770-e6ac09f7-465a-4c97-8dde-89734aa31e63.png)

**Comment by desht** [comment]
This crash could occur if the quest data hasn't been sync'd from the server, which should not normally happen. Normally the data is sync'd to the player when they log in, as long as their team data is correct. Could you check with the reporter that they are a member of a team (even a one person team, which they should be by default) ?  Also, any log data from the server at the point that player logged in (errors/stacktraces) would be really useful - it's possible a non-fatal error there prevented the quest data from syncing.

Next version of the mod will at least handle this gracefully and not crash, but it would be good to get to the bottom of why their quest data hasn't been sync'd.

**Comment by BlackAures1** [comment]
![image](https://user-images.githubusercontent.com/43911230/197199196-75a62681-75cc-4cea-b36f-f4b9b585beba.png)
This man is playing a single game as I understand it. Now I will ask him to write here. He will send the file today, I think.

**Comment by desht** [comment]
There will still be server logs playing single-player; the client and server logs are all in the client's `latest.log` in that case, since the client is using the integrated server.  So any information from that will be useful.

**Comment by BlackAures1** [comment]
Yes, I know. I'm just waiting for the person to be online, because the crash was happening to him, not me.

**Comment by stellarAnimiac** [comment]
Hello! I'm the one who managed to get outsmarted by a book and a button twice. 
It's been a while since I needed to retrieve the full log, 1.7.10 was the last time I needed to do so.
Would the full log still be in the files somewhere now that I've booted a few sessions after this? As it it wouldn't be the lastest log I mean. 
It was a really odd crash, just sort of out of the blue when it occured, and hasn't happened again, despite me using the menu much more since then.
